google/cloud/storage/retry.py

```diff
@@ -14,19 +14,21 @@
 
 import requests
 
-from google.api_core import exceptions
+from google.api_core import exceptions as exceptions_api
```

Comment:
Nit: maybe `api_exceptions` and `auth_exceptions`? Or just name the auth one and don't bother aliasing API.

Reply:
We call it `core_exceptions` in BQ